ports/nrf/boards/arduino_nano_33_ble/mpconfigboard.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2022 Arduino SA
+ */
+
+#define MICROPY_HW_BOARD_NAME       "Arduino Nano 33 BLE SENSE"
+#define MICROPY_HW_MCU_NAME         "NRF52840"
+#define MICROPY_PY_SYS_PLATFORM     "nano33"
```

Comment:
best to leave `MICROPY_PY_SYS_PLATFORM` unset, and let it take its default value of `nrf`